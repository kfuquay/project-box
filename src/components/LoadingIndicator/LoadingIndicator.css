.loading-container {
  text-align: center;
}

.spinner {
  color: #333;
}
