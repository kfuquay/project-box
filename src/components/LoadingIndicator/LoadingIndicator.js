import React from "react";
import "./LoadingIndicator.css";

function LoadingIndicator() {
  return (
    <div className="loading-container">
      <i className="fas fa-palette fa-spin" /> <p className="text">Loading...</p>
    </div>
  );
}

export default LoadingIndicator;
