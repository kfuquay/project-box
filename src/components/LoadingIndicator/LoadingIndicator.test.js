import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import LoadingIndicator from './LoadingIndicator';

describe(`LoadingIndicator component`, () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<LoadingIndicator/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
