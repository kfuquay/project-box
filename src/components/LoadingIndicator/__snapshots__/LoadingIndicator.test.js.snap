// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator component renders without crashing 1`] = `
<div
  className="loading-container main-container"
>
  <i
    className="fas fa-palette fa-spin spinner"
  />
   
  <p
    className="text"
  >
    Loading...
  </p>
</div>
`;
